cypress/support/commands/navigation.js

```diff
@@ -149,3 +149,29 @@ Cypress.Commands.add('selectRowByColumnAndValue', (columnName, expectedValue, mo
         });
     });
 });
+/**
+ * Select all rows on the current page
+ *
+ */
+Cypress.Commands.add('selectAllRowsOnCurrentPage', () => {
```

Comment:
I'd just rename this to `selectAllRows`.

Reply:
Wouldn't be a bit misleading? as I select all the rows on the current page, not all the rows from all the pages.